src/main/java/com/hotjava/app/hotjava/dto/Vehicle.java

```diff
@@ -19,7 +19,5 @@
     private String make;
     private String model;
     private int score;
-
-   // @OneToOne(mappedBy = "vehicle")
```

Comment:
Keep code uncommented in a seperate database branch to leave this code functionality in the program while still being able to develop other parts.